ClemBot.Bot/bot/clem_bot.py

```diff
@@ -140,40 +140,51 @@ async def command_claims_check(self, ctx: commands.Context):
         Before invoke hook to make sure a user has the correct claims to allow a command invocation
         """
         command = ctx.command
+
+        if command.ignore_claims_pre_invoke:
+            # The command is going to check the claims in the command body, nothing else to do
+            return 
+
+        if await self.claims_check(ctx) is True:
```

Comment:
`is True` is redundant
```suggestion
        if await self.claims_check(ctx):
```